tests/test_dimension.py

```diff
@@ -1103,15 +1107,17 @@ def test_no_fusion_convoluted(self):
 
         op = Operator(eqns)
 
-        exprs = FindNodes(Expression).visit(op._func_table['bf0'].root)
+        bns, _ = assert_blocking(op, {'x0_blk0', 'x1_blk0', 'x2_blk0'})
```

Comment:
OK now I'm a bit confused by the nomenclature. should it be `x0_blk0, x0_blk1, x0_blk2` Vince it's all the same `x` (same grid)

Reply:
Why is there two numbers actually?

Reply:
First number is per the old bf (nothing has changed)
i.e. `bf0 (x0_blk0...)`, `bf1 (x1_blk0...)` . 

The second number is about the blocking level (default one level)... Hierarchical blocking....

Reply:
```
x0_blk0
  x0_blk1
x1_blk0
  x1_blk1
    x1_blk2

```

Reply:
ok